devops/pipeline/cleanup_e2etest_infra.yaml

```diff
@@ -1,8 +1,16 @@
-# Test Environments tagged with environment: osconfig-e2etest should be removed daily
-# The 'osconfig-e2etest-debug' environment tag is used to persist the test infrastructure
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# 
+# End-to-end test environment cleanup pipeline
```

Comment:
>End-to-end test environment cleanup pipeline [](http://example.com/codeflow?start=2&length=44)

E2E tests should not be executed by the binary package pipeline but by the PR (aka: continuous integration) pipeline.

Can we accommodate this much needed change as part of this pipeline refactoring?

Reply:
I agree, most important change is the E2E tests.

Reply:
sure, ill put this as part of another PR